### ISSUE TITLE
[RFE] Add kubernetes-1.27 tests to the CI

### ISSUE THREAD

**Issue by tormath1** [kinvolk/Flatcar#999]
## Current situation

Currently, Kubernetes 1.27 is not tested in the CI on Flatcar (https://www.flatcar.org/docs/latest/container-runtimes/getting-started-with-kubernetes/#supported-kubernetes-version).

## Implementation options

* It's possible to take inspiration from this PR: https://github.com/flatcar/mantle/pull/406
* Use the [`checksum.sh`](https://github.com/flatcar/mantle/blob/flatcar-master/kola/tests/kubeadm/checksums.sh) helper
* Once the Mantle test is merged and it works correctly, open a follow-up PR to add Kubernetes 1.27 to the list of _officially_ supported Kubernetes version (https://www.flatcar.org/docs/latest/container-runtimes/getting-started-with-kubernetes/#supported-kubernetes-version)

Note: it might be interesting to test this new feature https://kubernetes.io/docs/concepts/cluster-administration/system-logs/#log-query as it relies on systemd-journald from the OS

**Comment by msitworld** [comment]
Hi
Is there any update about this issue? Does Flatcar support Kubernetes 1.27?
Kubernetes 1.28 will be released on the 15th of August too.

**Comment by tormath1** [comment]
Hi @msitworld - Kubernetes 1.27 should work on Flatcar, this issue is just to track the addition of 1.27 testing in Flatcar's CI. Let us know if you have any issues with 1.27 in your testing :)
